common/pom.xml

```diff
@@ -77,6 +77,12 @@
       <artifactId>log4j-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
@aniketbhatnagar can we get rid of this ? I would prefer to not add another dependency for a few tests.